src/BookReader.js

```diff
@@ -662,93 +673,146 @@ BookReader.prototype.resize = function() {
 };
 
 /**
- * Binds keyboard event listeners
+ * Intersection observer callback sets BookReader keyboard focus
+ * off when the BookReader is not in the viewport.
+ * @param {object} entries
+ * @param {function} observer
  */
-BookReader.prototype.setupKeyListeners = function() {
-  var self = this;
+BookReader.prototype.handleIntersection = function (entries, observer) {
+  entries.forEach((entry) => {
+    if (entry.intersectionRatio === 0) {
+      this.hasKeyFocus = false;
+    }
+  });
+};
 
-  var KEY_PGUP = 33;
-  var KEY_PGDOWN = 34;
-  var KEY_END = 35;
-  var KEY_HOME = 36;
-
-  var KEY_LEFT = 37;
-  var KEY_UP = 38;
-  var KEY_RIGHT = 39;
-  var KEY_DOWN = 40;
-  // The minus(-) and equal(=) keys have different mappings for different browsers
-  var KEY_MINUS = 189; // Chrome
-  var KEY_MINUS_F = 173; // Firefox
-  var KEY_NUMPAD_SUBTRACT = 109;
-  var KEY_EQUAL = 187; // Chrome
-  var KEY_EQUAL_F = 61; // Firefox
-  var KEY_NUMPAD_ADD = 107;
-
-  // We use document here instead of window to avoid a bug in jQuery on IE7
-  $(document).on("keydown", function(e) {
-
-    // Keyboard navigation
-    switch (e.keyCode) {
-    case KEY_PGUP:
-    case KEY_UP:
-      // In 1up mode page scrolling is handled by browser
-      if (!utils.isInputActive() && self.constMode2up == self.mode) {
-        e.preventDefault();
-        self.prev();
-      }
+/**
+ * Returns true if element in area to activate BookReader keyboard focus
+ * @param {object} element
+ * @returns {boolean}
+ */
+BookReader.prototype.isKeyFocus = function (element) {
+  if (element.closest('#theatre-ia')) {
+    return true;
+  } else if (element.closest('#topnav')) {
+    return true;
+  }
+  return false;
```

Comment:
Ah, this is a little non-ideal ; is this why you were asking about how to test standalone BR? This seems like it might be the simplest solution:

```suggestion
  return Boolean(element.closest('#theatre-ia, #topnav, ia-bookreader, #BookReader'));
```

Longer term, this should only need `ia-bookreader`, but we're not there yet.

Reply:
I'm confused about `#topnav` ; I think you mentioned it earlier, but I didn't fully understand. Can you expand as to why that's needed?

Reply:
Thanks, re-read https://developer.mozilla.org/en-US/docs/Web/API/Element/closest about selector list.

> `#topnav`
If:
- clicking in
- tabbing to controls 
in TopNav area, then so long as one doesn't activate them, i.e., leave the page, one is ... sensibly ... in BookReader, rather than "page" focus.

That is, simply, a narrative use case that goes like this: 
"I've navigated to a book, and I see some stuff at the top. I'll click something, for example, the user icon, and see a drop down menu. When I don't choose anything in that menu, I expect BookReader to have keyboard shortcut focus."